doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -84,6 +84,22 @@ kieservers:
 ----
 +
 You can use different configuration sources in any other configuration parameter. For more information, see the https://quarkus.io/guides/config-reference#configuration-sources[CONFIGURATION REFERENCE GUIDE].
+. Optional: To configure Mutual TLS (mTLS) authentication, that uses client certificates rather than credentials or tokens, configure `client-cert` for mTLS, for example:
```

Comment:
mTLS does not exclude the use of tokens or user/password authentication. This sentence can bring to confusion. Users can use mTLS or mTLS + [token|basic auth] or only [token|basic auth]